packages/node_modules/@webex/plugin-meetings/src/metrics/index.js

```diff
@@ -433,16 +432,28 @@ class Metrics {
 
     const tags = {
       ...metricTags,
+      browser: bowser.name,
       org_id: this.webex.credentials.getOrgId(),
-      os: bowser.osname,
-      platform: bowser.name
+      os: bowser.osname
     };
 
     if (!metricName) {
       throw Error('Missing operational metric name. Please provide one');
     }
 
-    sendMetric(metricName, {type: ['operational'], fields, tags});
+    this.webex.request({
```

Comment:
I'm a bit nervous about making this change away from `sendMetric()`. Do we happen to know why we cannot use the `sendMetric()` method? As in, do we have the specific error message being generated? I can attempt to chase this cause a bit in the meantime.

Reply:
Is there something specific about this change that makes you nervous?
This is doing basically what the metrics plugin sans the batching.

The issue is that metric's `submitClientMetrics` relies on other functions within the plugin, but once it's imported here, those other functions are not there.